consul/README.md

```diff
@@ -102,17 +102,52 @@ Collecting logs is disabled by default in the Datadog Agent. To enable it, see [
 
 #### DogStatsD
 
-Alternatively, you can configure Consul to send data to the Agent through [DogStatsD][3] instead of relying on the Agent to pull the data from Consul. To achieve this, add your `dogstatsd_addr` nested under the top-level `telemetry` key in the main Consul configuration file:
+Optionally, you can configure Consul to also send data to the Agent through [DogStatsD][3] instead of relying on the Agent to pull the data from Consul. 
 
-```conf
-{
-  ...
-  "telemetry": {
-    "dogstatsd_addr": "127.0.0.1:8125"
-  },
-  ...
-}
-```
+1. Configure Consul to send DogStatsD metrics by adding the `dogstatsd_addr` nested under the top-level `telemetry` key in the main Consul configuration file:
+
+    ```conf
+    {
+      ...
+      "telemetry": {
+        "dogstatsd_addr": "127.0.0.1:8125"
+      },
+      ...
+    }
+    ```
+
+1. Update the [Datadog Agent main configuration file][16] `datadog.yaml` by adding the following configs to ensure metrics are tagged correctly:
+
+   ```yaml
+   # dogstatsd_mapper_cache_size: 1000  # default to 1000
```

Comment:
This might be out of scope, but do you know if adding this to our existing e2e will be possible?

Reply:
That's partially addressed in https://github.com/DataDog/integrations-core/pull/6777 to setup the agent side.  Can add another config to Consul as a separate PR though.